.cspell.json

```diff
@@ -0,0 +1,109 @@
+{
+    "version": "0.1",
+    "words": [
+        "alignof",
+        "apikey",
+        "badterms",
+        "bdsw",
+        "Beihang",
+        "blinenums",
+        "blocklist",
+        "capi",
+        "charmod",
+        "Cmembership",
+        "constexpr",
+        "Cprocess",
+        "cssvalidation",
+        "customised",
+        "dahut",
+        "DAHUT",
+        "decltype",
+        "docdate",
+        "doctypes",
+        "editorsid",
+        "elif",
+        "ERCIM",
+        "esac",
+        "FPIG",
+        "FPWD",
+        "FPWG",
+        "gzipped",
+        "hellip",
+        "hilite",
+        "ifdef",
+        "ifndef",
+        "ignote",
+        "Imple",
+        "implelink",
+        "imsc",
+        "insafe",
+        "isnt",
+        "Keio",
+        "ldquo",
+        "linenums",
+        "linkchecker",
+        "linkhref",
+        "Localise",
+        "mediacapture",
+        "memsub",
+        "metaviewport",
+        "middot",
+        "munge",
+        "newurl",
+        "nodate",
+        "nonlocal",
+        "Normalise",
+        "Nresult",
+        "nullptr",
+        "OBSL",
+        "ocument",
+        "ontents",
+        "patentpolicy",
+        "Pubrules",
+        "rdquo",
+        "recognising",
+        "rectrack",
+        "Resc",
+        "RSCND",
+        "sbyte",
+        "secno",
+        "secnos",
+        "sels",
+        "shortlink",
+        "shortname",
+        "shortnames",
+        "sotd",
+        "sparql",
+        "specberus",
+        "SPSD",
+        "stackalloc",
+        "strictfp",
+        "subm",
+        "supersedable",
```

Comment:
what does this word mean? `super stable`?

Reply:
Something that can be replaced/overridden by something else https://www.merriam-webster.com/dictionary/supersedable